src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -534,8 +564,13 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         console.error(error);
         /* eslint-enable no-console */
 
-        // eslint-disable-next-line react/prop-types
-        this.props.onError(error, code, contextInfo);
+        const { onAnnotationsFetchError, onError } = this.props;
+
+        onError(error, code, contextInfo);
+
+        if (code === ERROR_CODE_FETCH_ANNOTATIONS) {
+            onAnnotationsFetchError(error);
```

Comment:
Should this live in the fetchAnnotations error callback, instead of in the global handler?

Reply:
There is no fetchAnnotations error callback. Annotations are fetch as part of `feedItems` api call. We can only use this global error handler.